docs/content/SRS/SRS.md

```diff
@@ -261,23 +379,45 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 15.1 Access Requirements
 
-[Insert your content here.]
+SR-AC1. The application will use secure environment variables to manage configuration settings that include sensitive information.
+
+Rationale: Managing settings through environment variables keeps sensitive information out of the codebase, reducing the risk of exposure and enhancing security.
 
 ### 15.2 Integrity Requirements
 
-[Insert your content here.]
+SR-INT1. All communication between the client UI, backend services, and external APIs must be encrypted using HTTPS.
+
+Rationale: HTTPS encryption secures data in transit, preventing interception or tampering. It also ensures the confidentiality and integrity of user data and commands.
+
+SR-INT2. Implement DNS security measures to ensure that DNS queries and responses are protected against tampering and spoofing.
+
+Rationale: Securing DNS interactions prevents attackers from manipulating or rerouting network traffic. This is critical for maintaining the integrity of application data.
+
+SR-INT3. The application will use content security policies to mitigate the risk of XSS attacks.
```

Comment:
let's also include a section for implement JWT and short-lived tokens to ensure communication during sessions are secured.

Think about a long planning session, with users notes. We def need to ensure bad actors won't be able to intercept :)

Reply:
Included requirement for implementing JWT and short-lived tokens under integrity requirements(SR-INT4).